### PR TITLE
Add upcoming night maps

### PR DESCRIPTION
**Pull request by cemathey**
Add the new map names for the upcoming existing map night variants.

I confirmed that the existing night maps share RCON game log map names with their day variants, for example this is actually Kursk Night:

`11:00:10 - Fri, Oct 13	MATCH START         MATCH START KURSK WARFARE`

### CODE REVIEW

**Review by zMoooooritz — APPROVED** [review]

File: rcon/utils.py [hunk 1]
```diff
@@ -163,33 +172,39 @@ def get_map_side(map_):
...
     "kharkov_offensive_rus": "Kharkov Offensive (RUS)",
+    "kharkov_warfare_night": "Kharkov (Night)",
     "kharkov_warfare": "Kharkov",
     "kursk_offensive_ger": "Kursk Offensive (GER)",
     "kursk_offensive_rus": "Kursk Offensive (RUS)",
     "kursk_warfare_night": "Kursk (Night)",
     "kursk_warfare": "Kursk",
     "omahabeach_offensive_ger": "Omaha Beach Offensive (GER)",
     "omahabeach_offensive_us": "Omaha Beach Offensive (US)",
+    "omahabeach_warfare_night": "Omaha (Night)",
```
Comment (timraay) [inline rcon/utils.py:207]: For consistency, this should say "Omaha Beach". Also, while "night" is the safe option here, it's technically supposed to be a "dusk" map? Same for El Alamein. Maybe something to remember for the future.

File: rcon/utils.py [hunk 2]
```diff
@@ -201,48 +216,57 @@ def get_map_side(map_):
...
     "stmariedumont_off_ger": "Ste Marie du Mont Offensive (GER)",
     "stmariedumont_off_us": "Ste Marie du Mont Offensive (US)",
     "stmariedumont_warfare": "Ste Marie du Mont",
     "stmereeglise_offensive_ger": "Ste Mère Église Offensive (GER)",
     "stmereeglise_offensive_us": "Ste Mère Église Offensive (US)",
+    "stmereeglise_warfare_night": "Ste Mère Église (Night)",
     "stmereeglise_warfare": "Ste Mère Église",
     "utahbeach_offensive_ger": "Utah Beach Offensive (GER)",
     "utahbeach_offensive_us": "Utah Beach Offensive (US)",
+    "utahbeach_warfare_night": "Utah (Night)",
```
Comment (timraay) [inline rcon/utils.py:230]: Same here, should be "Utah Beach"

File: rcon/utils.py [hunk 3]
```diff
@@ -307,48 +340,57 @@ def get_map_side(map_):
...
     "stmariedumont_off_ger": "Ste Marie du Mont (GER)",
     "stmariedumont_off_us": "Ste Marie du Mont (US)",
     "stmariedumont_warfare": "Ste Marie du Mont",
     "stmereeglise_offensive_ger": "Ste Mère Église (GER)",
     "stmereeglise_offensive_us": "Ste Mère Église (US)",
+    "stmereeglise_warfare_night": "Ste Mère Église (Night)",
     "stmereeglise_warfare": "Ste Mère Église",
     "utahbeach_offensive_ger": "Utah Beach (GER)",
     "utahbeach_offensive_us": "Utah Beach (US)",
+    "utahbeach_warfare_night": "Utah (Night)",
```
Comment (timraay) [inline rcon/utils.py:354]: Utah Beach

File: rcon/utils.py [hunk 4]
```diff
@@ -254,48 +278,57 @@ def get_map_side(map_):
...
     "kharkov_offensive_rus": "Kharkov (RUS)",
+    "kharkov_warfare_night": "Kharkov (Night)",
     "kharkov_warfare": "Kharkov",
     "kursk_offensive_ger": "Kursk (GER)",
     "kursk_offensive_rus": "Kursk (RUS)",
     "kursk_warfare_night": "Kursk (Night)",
     "kursk_warfare": "Kursk",
     "omahabeach_offensive_ger": "Omaha (GER)",
     "omahabeach_offensive_us": "Omaha Beach (US)",
+    "omahabeach_warfare_night": "Omaha (Night)",
```
Comment (timraay) [inline rcon/utils.py:331]: Omaha Beach

File: rcon/utils.py [hunk 5]
```diff
@@ -307,48 +340,57 @@ def get_map_side(map_):
...
     "elalamein_offensive_ger": "El Alamein (GER)",
+    "elalamein_warfare_night": "El Alamein (Night)",
     "elalamein_warfare": "El Alamein",
     "foy_offensive_ger": "Foy (GER)",
     "foy_offensive_us": "Foy (US)",
     "foy_warfare_night": "Foy (Night)",
     "foy_warfare": "Foy",
     "hill400_offensive_ger": "Hill400 (GER)",
     "hill400_offensive_US": "Hill400 (US)",
+    "hill400_warfare_night": "Hill 400 (Night)",
```
Comment (timraay) [inline rcon/utils.py:377]: Hill400

**Review by timraay — APPROVED** [review]

**Pull request merged**